roles/openshift_hosted/tasks/registry/registry.yml

```diff
@@ -40,6 +40,9 @@
     openshift_hosted_registry_images: "{{ openshift.hosted.registry.registryurl | default('openshift3/ose-${component}:${version}')}}"
     openshift_hosted_registry_volumes: []
     openshift_hosted_registry_env_vars: {}
+    openshift_hosted_registry_route_certificates: "{{ openshift.hosted.registry.route.certificates | default(false) }}"
+    openshift_hosted_registry_route_host: "{{ openshift.hosted.registry.route.host | default(false) }}"
```

Comment:
@kwoodson that's why I'm confused about why we have two different variables. If we are deprecating docker_registry_route_hostname, then I don't see why doing something like this wouldn't work:

in vars/main.yml
```
osh_default_subdomain: router.default.svc.cluster.local
oshr_default_route_host: "{{ 'docker-registry-default.' ~ (openshift_master_default_subdomain | default(default_subdomain, true) }}"
```
in defaults/main.yml
```
openshift_hosted_registry_route_host: "{{ docker_registry_route_hostname | oshr_default_route_host }}"
```

Then tasks/registry/secure.yml could be updated to:
- remove the `set_fact` for `docker_registry_route_hostname`
- use `openshift_hosted_registry_route_host` in place of `docker_registry_route_hostname`

Granted, this doesn't take into account insecure registries, but I think you could conditionalize the default value for that:
```
oshr_secure: "{{ not openshift.docker.hosted_registry_insecure | default(false) }}"
osh_default_subdomain: router.default.svc.cluster.local
oshr_default_route_host_secure: "{{ 'docker-registry-default.' ~ (openshift_master_default_subdomain | default(default_subdomain, true) }}"
oshr_default_route_host: "{{ oshr_default_route_host_secure if oshr_secure else None }}" 
```